src/stories/Header.stories.jsx

```diff
@@ -0,0 +1,46 @@
+import Header from "./Header";
```

Comment:
This is an incorrect implementation of story book. With this implementation we can just see the header but we cannot customise it in storybook. 

Run the storybook using `yarn storybook` or similar command and play with `Button` component you will notice that it allows to control things like its UI, text and many more things within the storybook we want something like that.

Reply:
I fixed it by adding different arguments of storybook in different variations.
![image](https://github.com/user-attachments/assets/dfb524dc-b17f-4879-a0b2-4b0aca1daa92)